### PR TITLE
Issues/#27

### PR DESCRIPTION
**Pull request by kimuray**
企業が求人を出せる機能を作成しました。
確認お願いいたします。

### CODE REVIEW

File: app/views/offers/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <th>Deadline</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @offers.each do |offer| %>
+      <tr>
+        <td><%= offer.title %></td>
+        <td><%= offer.company.name %>
```
Comment (mineko-fuji) [inline app/views/offers/index.html.erb]: __[rails_best_practices]__
* law of demeter [:ref](http://rails-bestpractices.com/posts/2010/07/24/the-law-of-demeter/)

File: app/controllers/offers_controller.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
+class OffersController < InheritedResources::Base
```
Comment (mineko-fuji) [inline app/controllers/offers_controller.rb]: __[reek]__
* [IrresponsibleModule] has no descriptive comment [:ref](https://github.com/troessner/reek/blob/master/docs/Irresponsible-Module.md)

File: app/helpers/offers_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module OffersHelper
```
Comment (mineko-fuji) [inline app/helpers/offers_helper.rb]: __[rails_best_practices]__
* remove empty helpers [:ref](http://rails-bestpractices.com/posts/2011/04/09/remove-empty-helpers/)

File: app/controllers/offers_controller.rb [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  
+  # POST /companies/1/offers
+  # POST /companies/1/offers.json
+  def create
+    @offer = Offer.new(offer_params)
+    @offer.company_id = current_user.company_id
+
+    respond_to do |format|
+      if @offer.save
+        format.html { redirect_to offers_path, notice: 'Offer was successfully created.' }
```
Comment (mineko-fuji) [inline app/controllers/offers_controller.rb]: __[reek]__
* [DuplicateMethodCall] calls format.html 2 times [:ref](https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md)

File: app/views/offers/_form.html.erb [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<%= form_for(@offer) do |f| %>
```
Comment (mineko-fuji) [inline app/views/offers/_form.html.erb]: __[rails_best_practices]__
* replace instance variable with local variable [:ref](http://rails-bestpractices.com/posts/2010/07/24/replace-instance-variable-with-local-variable/)

File: app/controllers/offers_controller.rb [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+        format.json { render :show, status: :created, location: @offer }
+      else
+        format.html { render :new }
```
Comment (mineko-fuji) [inline app/controllers/offers_controller.rb]: __[reek]__
* [DuplicateMethodCall] calls format.html 2 times [:ref](https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md)

File: app/views/offers/_form.html.erb [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+  <% if @offer.errors.any? %>
```
Comment (mineko-fuji) [inline app/views/offers/_form.html.erb]: __[rails_best_practices]__
* replace instance variable with local variable [:ref](http://rails-bestpractices.com/posts/2010/07/24/replace-instance-variable-with-local-variable/)

File: app/controllers/offers_controller.rb [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+  # POST /companies/1/offers
+  # POST /companies/1/offers.json
+  def create
+    @offer = Offer.new(offer_params)
+    @offer.company_id = current_user.company_id
+
+    respond_to do |format|
+      if @offer.save
+        format.html { redirect_to offers_path, notice: 'Offer was successfully created.' }
+        format.json { render :show, status: :created, location: @offer }
```
Comment (mineko-fuji) [inline app/controllers/offers_controller.rb]: __[reek]__
* [DuplicateMethodCall] calls format.json 2 times [:ref](https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md)

File: app/views/offers/_form.html.erb [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+    <div id="error_explanation">
+      <h2><%= pluralize(@offer.errors.count, "error") %> prohibited this offer from being saved:</h2>
```
Comment (mineko-fuji) [inline app/views/offers/_form.html.erb]: __[rails_best_practices]__
* replace instance variable with local variable [:ref](http://rails-bestpractices.com/posts/2010/07/24/replace-instance-variable-with-local-variable/)

File: app/controllers/offers_controller.rb [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+      else
+        format.html { render :new }
+        format.json { render json: @offer.errors, status: :unprocessable_entity }
```
Comment (mineko-fuji) [inline app/controllers/offers_controller.rb]: __[reek]__
* [DuplicateMethodCall] calls format.json 2 times [:ref](https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md)

File: app/views/offers/_form.html.erb [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+
+      <ul>
+      <% @offer.errors.full_messages.each do |message| %>
```
Comment (mineko-fuji) [inline app/views/offers/_form.html.erb]: __[rails_best_practices]__
* replace instance variable with local variable [:ref](http://rails-bestpractices.com/posts/2010/07/24/replace-instance-variable-with-local-variable/)

File: app/controllers/offers_controller.rb [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+    @offer = Offer.new
+  end
+  
+  # GET /companies/1/offers/1/edit
+  def edit
+  end
+  
+  # POST /companies/1/offers
+  # POST /companies/1/offers.json
+  def create
```
Comment (mineko-fuji) [inline app/controllers/offers_controller.rb]: __[reek]__
* [TooManyStatements] has approx 11 statements [:ref](https://github.com/troessner/reek/blob/master/docs/Too-Many-Statements.md)

Comment (mineko-fuji) [inline app/controllers/offers_controller.rb]: __[rails_best_practices]__
* use model association (for @offer) [:ref](http://rails-bestpractices.com/posts/2010/07/19/use-model-association/)

File: app/controllers/offers_controller.rb [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+      else
+        format.html { render :edit }
+        format.json { render json: @offer.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /companies/1/offers/1
+  # DELETE /companies/1/offers/1.json
+  def destroy
```
Comment (mineko-fuji) [inline app/controllers/offers_controller.rb]: __[reek]__
* [TooManyStatements] has approx 6 statements [:ref](https://github.com/troessner/reek/blob/master/docs/Too-Many-Statements.md)

File: app/controllers/offers_controller.rb [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+      end
+    end
+  end
+  
+  # PATCH/PUT /companies/1
+  # PATCH/PUT /companies/1.json
+  def update
+    respond_to do |format|
+      if @offer.update(offer_params)
+        format.html { redirect_to offers_path, notice: 'Offer was successfully updated.' }
```
Comment (mineko-fuji) [inline app/controllers/offers_controller.rb]: __[reek]__
* [DuplicateMethodCall] calls format.html 2 times [:ref](https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md)

File: app/controllers/offers_controller.rb [hunk 15]
```diff
@@ -0,0 +1,74 @@
...
+        format.json { render :show, status: :ok, location: @offer }
+      else
+        format.html { render :edit }
```
Comment (mineko-fuji) [inline app/controllers/offers_controller.rb]: __[reek]__
* [DuplicateMethodCall] calls format.html 2 times [:ref](https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md)

File: app/controllers/offers_controller.rb [hunk 16]
```diff
@@ -0,0 +1,74 @@
...
+    end
+  end
+  
+  # PATCH/PUT /companies/1
+  # PATCH/PUT /companies/1.json
+  def update
+    respond_to do |format|
+      if @offer.update(offer_params)
+        format.html { redirect_to offers_path, notice: 'Offer was successfully updated.' }
+        format.json { render :show, status: :ok, location: @offer }
```
Comment (mineko-fuji) [inline app/controllers/offers_controller.rb]: __[reek]__
* [DuplicateMethodCall] calls format.json 2 times [:ref](https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md)

File: app/controllers/offers_controller.rb [hunk 17]
```diff
@@ -0,0 +1,74 @@
...
+      else
+        format.html { render :edit }
+        format.json { render json: @offer.errors, status: :unprocessable_entity }
```
Comment (mineko-fuji) [inline app/controllers/offers_controller.rb]: __[reek]__
* [DuplicateMethodCall] calls format.json 2 times [:ref](https://github.com/troessner/reek/blob/master/docs/Duplicate-Method-Call.md)

File: app/controllers/offers_controller.rb [hunk 18]
```diff
@@ -0,0 +1,74 @@
...
+      else
+        format.html { render :new }
+        format.json { render json: @offer.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+  
+  # PATCH/PUT /companies/1
+  # PATCH/PUT /companies/1.json
+  def update
```
Comment (mineko-fuji) [inline app/controllers/offers_controller.rb]: __[reek]__
* [TooManyStatements] has approx 9 statements [:ref](https://github.com/troessner/reek/blob/master/docs/Too-Many-Statements.md)

**Comment by lasershow** [comment]
OKです　mergeしましょう！

**Pull request merged**